spec/index.adoc

```diff
@@ -3614,7 +3621,7 @@ Params = {
   reject_message : Text;
   sysenv : Env;
   cycles_refundend : Nat;
-  method_name : Text;
+  method_name : NoData | Text;
```

Comment:
I wonder why these aren't `Optional Text` or `Text?`?

Reply:
That's roughly what it means, but without the extra constructor in the non-null case to deal with everywhere. In a mechanization, I'd use an option